helpers/foundation-deployer/stages/data.go

```diff
@@ -245,6 +246,10 @@ type NetSharedTfvars struct {
 	TargetNameServerAddresses []ServerAddress `hcl:"target_name_server_addresses"`
 }
 
+type NetProductionTfvars struct {
```

Comment:
this is a required field, not an optional field, what happens when you use the helper to deploy the hub and spoke alternative?